spec/models/user_spec.rb

```diff
@@ -58,4 +58,24 @@
       its(:opted_in?) { should be_falsey }
     end
   end
+
+  describe '#admin?' do
+    context 'user has an education.gov.uk email' do
+      subject { create(:user, email: 'test@education.gov.uk') }
```

Comment:
```suggestion
      subject { create(:user, :admin }
```

Reply:
I think we're testing the implementation here, so we should hard code it. The trait is useful for other areas where we need an admin.